lib/process-services-cloud/src/lib/people/components/people-cloud.component.ts

```diff
@@ -275,6 +281,13 @@ export class PeopleCloudComponent implements OnInit, OnChanges, OnDestroy {
         return false;
     }
 
+    private isExcludedUser(searchUser: IdentityUserModel): boolean {
+        if (this.excludedUsers && this.excludedUsers.length > 0) {
```

Comment:
can we use `if (this.excludedUsers?.length > 0)` and declare a local variable isExcluded for the result?
Something like:

```
let isExcluded = false;
if (this.excludedUsers?.length > 0) {
isExcluded = !!this.excludedUsers.find(excludedUser => this.compare(excludedUser, searchUser));
}
return isExcluded;
```

it would make debugging easier

Reply:
I'm not sure how will will make the debugging easier by creating a local variable.
Also I don't think using a local variable will make any sense in this function since this function has only one condition and the debugging purpose is not enough to add it in my opinion.

Reply:
I personally think that having multiple exits on a function makes it more difficult to understand and debug but that's not a rule it is just my personal opinion. Please take a look at this article https://www.tomdalling.com/blog/coding-tips/coding-tip-have-a-single-exit-point/ and proceed as you wish

Reply:
Although I do agree, having a single exit point is considered to be a good practice with (especially complex) functions, which can help making the code easier to understand, however with such a simple function (which consists of 4 lines), I think the readability and understandability isn't damaged.

I agree with using the optional chaining, it is essentially the same, but less code:
```
this.excludedUsers?.length > 0
```